understanding/21/text-spacing.html

```diff
@@ -18,7 +18,7 @@ <h3 id="authorresp">Author Responsibility </h3>
          <p>Further, this SC is not concerned with <em>how</em> users change the line height and spacing metrics. It does not require that content implement its own mechanisms to allow users to do this. It is not a failure of the content if a user agent or platform does not provide a way for users to do this. Content does not fail this SC if the method chosen by the user - for instance, the use of an extension or bookmarklet - fails to correctly set the line height and spacing text properties on the content (provided that the content is not actively and purposely preventing the properties from being added).</p>
          <section>
          <h4 id="applicability">Applicability</h4>
-         <p>If the markup-based technologies being used are capable of overriding text to the Success Criterion's metrics, then this SC is applicable. For instance Cascading Style Sheet/HTML technologies are quite able to allow for the specified spacing metrics. Plugin technologies would need to have a built-in ability to modify styles to the specified metrics. Currently, this SC does not apply to PDF as it is not implemented using markup.</p>
+         <p>If the markup-based technologies being used are capable of overriding text to the Success Criterion's metrics, then this SC is applicable. For instance, Cascading Style Sheet/HTML technologies generally allow for the specified spacing metrics. However, if discrete portions of a markup language do not support any of the listed text style properties, then it is not applicable to that portion of the content. For example, Web Components are a part of HTML, but do not support modification of any of the listed text style properties if they have been implemented with a closed ShadowRoot. Such Web Components and other plugin technologies would need to have a built-in ability to modify styles to the specified metrics. Currently, this SC does not apply to PDF as it is not implemented using markup.</p>
```

Comment:
per the wg call today, this note already states that the closed shadowRoot web components would need to have a built-in ability to modify styles.  So I (incorrectly) thought that would have addressed the concerns that people were raising, since it's still staying that there would need to be a built-in way to adjust?

would appreciate feedback/suggestions on what could be clarified here - or is it that the note needs to more overtly state that if an author can avoid closed shadowroot, then they are recommended to go that route. And that maybe there needs to be more overt call outs to certain HTML features (provided by browsers) that do use this and cannot be changed by authors, and those should arguably be ok?  e.g., the date picker exapmle?